system-int-test/test/cmr/system_int_test/search/collection_eula_json_format_test.clj

```diff
@@ -0,0 +1,30 @@
+(ns cmr.system-int-test.search.collection-eula-json-format-test
+  (:require
+   [clojure.test :refer :all]
+   [cmr.common.util :as util]
+   [cmr.system-int-test.data2.core :as data2]
+   [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.ingest-util :as ingest]
+   [cmr.system-int-test.utils.search-util :as search]))
+
+(use-fixtures :each (ingest/reset-fixture {"provguid1" "PROV1"}))
+
+(deftest search-collections-eula-identifiers
+  (let [coll1 (data2/ingest-umm-spec-collection "PROV1"
+                                                (data-umm-c/collection 1
+                                                                       {:EntryTitle "Dataset1"
+                                                                        :UseConstraints {:Description "Eula"
+                                                                                         :LicenseURL {:Linkage "https://somelicenseurl.org",
+                                                                                                      :Name "License URL",
+                                                                                                      :Description "This is the License URL for this data set.",
```

Comment:
Do we have a better distinction for "license" since there could also be "useConstraints"?

Reply:
I'm not entirely sure.  It is a required field, which is why I added it here.

Reply:
Its pulled directly from test data found in SIT
```
            <UseConstraints>
                <Description>There are no use constraints or legal constraints in using this data set.</Description>
                <EULAIdentifier>EchoId1</EULAIdentifier>
                <EULAIdentifier>EchoId2</EULAIdentifier>
                <EULAIdentifier>EchoId3</EULAIdentifier>
                <LicenseURL>
                    <URL>https://somelicenseurl.org</URL>
                    <Description>This is the License URL for this data set.</Description>
                    <Type>License URL</Type>
                    <MimeType>text/html</MimeType>
                </LicenseURL>
            </UseConstraints>
```